docs/UserGuide.adoc

```diff
@@ -97,23 +122,14 @@ Format: `add MODULE_CODE`
 [TIP]
 Modules are imported from NUSMods database.
 
-
 Examples:
 
 * `add cs2103t`
 
-=== Viewing a module: `view`
-
-Views a module and enters the module view. Can be a tracked module or an archived module.
-
-Format: `view <MODULE_CODE>`
-
-Example: `view cs1231`
-
-=== Return from module view: `back`
-Returns from the module view.
+=== Deleting a module: `delete`
 
-Format: `back`
+Untracks a module. +
+Format: `delete <index>`, index is the index of the tracked module displayed on the list through the `list` command.
```

Comment:
Delete no longer uses index, module code is used instead. Maybe add an example as well.